docs/modules/servers/pages/index.adoc

```diff
@@ -0,0 +1,92 @@
+= Apache James Servers
+:navtitle: Servers
+
+James offers a number of ready-made servers. The servers are intended for those
+with basic needs, and can be used out-of-the-box. These servers have been
+tested and verified so you can get set up quickly with a production-grade
+mail server that you can use with confidence.
+
+The available James Servers are:
+
+ * <<demo,James Demo Server>>
+ * <<minimal,James Minimal Server>>
+ * <<basic,James Basic Server>>
+ * <<advanced,James Advanced Server>>
+ * <<distributed,James Distributed Server>>
```

Comment:
IMO we should have a goal to 'limit' servers that we provide.

More servers = more work.

I make little distinction between minial & basic and advanced & distributed, that would get implemented by the way currently by the same technologies.

Reply:
Yeah, not a bad idea.

So what about offering only these 3:

 * Demo
 * Basic
 * Advanced

As was pointed out by @rouazana, both Basic and Advanced could have different "Flavours".

The advantage of having only 3 servers is that it would be much easier to understand for the users. It would likely be much less confusing.

If the "flavours" are eliminated, it would actually be less work for the James team. So maybe just keep what is currently the "distributed" flavour for advanced, and a relatively simple server for the Basic server (not distributed, just the basic email features that most users would expect).

Could that work?

Reply:
Yes I think so :-)

Reply:
Could these "flavours" be configured somehow? If I were a user who wanted to set up a "Basic Server" with, say, the filesystem instead of RDB (I know it's not supported, but just for the sake of an example), how would I, as the user, configure my "Basic Server" to get this result?

Reply:
Note: I have updated this part as suggested by @rouazana to use {Demo, Local, Redundant, Distributed} instead of {Demo, Minimal, Basic, Advanced, Distributed}.